### PR TITLE
CHANGELOG updates for 2022.12

### PR DESCRIPTION
**Pull request by tobiasd**
This shall be the last PR merged into the 2022.12-rc branch before the release of the 2022.12 stable version of Friendica

### CODE REVIEW

**Review by MrPetovan — APPROVED** [review]

File: CHANGELOG [hunk 1]
```diff
@@ -1,13 +1,65 @@
...
+    Added a moderation section to the admin panel [annando]
+    Added an option to make the calendar public [matthiasmoritz]
+    Fixed a bug in the federation with Diaspora* [annando]
+    Deprecated old themes (duepuntozero, quattro, smoothy)
+    
+    NOTE: The Apache2 rewrite rule in the .htaccess-dist has been changed.
+          The change has to be applied manually to the existing .htaccess.
 
   Friendica Addons
+    Updates to the translations DE, FR, SV
```
Comment (urbalazs) [inline CHANGELOG]: @tobiasd you can add HU after pulling the latest translations from Transifex

**Review by MrPetovan — APPROVED** [review]

**Comment by MarekBenjamin** [comment]
What about the image grid? 👼

**Comment by tobiasd** [comment]
> What about the image grid? angel

I have it in the highlights for the release, but missed it in the CHANGELOG... xD

**Review by MrPetovan — APPROVED** [review]

**Comment by foss-** [comment]
Late to the party, but can we highlight that frio hit v1.0? Currently it goes under in the "some fixes in themes" point:
`Updates to the themes (frio, vier) [loma-one, MarekBenjamin, MrPetovan, tobiasd]`
Maybe something more enthusiastic?
`frio, our default friendica theme, sees many improvements and fixes and reaches v1.0`

**Comment by tobiasd** [comment]
> Late to the party, but can we highlight that frio hit v1.0?

My question is more, why wasn't it made 1.0 when we used it as default theme? It should have been in my opinion. But I can mention it in the release notes more prominently.

**Pull request merged**